src/ctapipe/reco/impact.py

```diff
@@ -85,51 +90,59 @@ class ImPACTReconstructor(Component):
 
     """
 
+    minimiser = traits.CaselessStrEnum(
+        ["minuit", "l-BFGS"],
```

Comment:
Any point in supporting the scipy minimize?

Reply:
No, I think that can be removed